lib/autocomplete.js

```diff
@@ -0,0 +1,90 @@
+const _ = require("lodash");
+const aws = require("aws-sdk");
+const parsers = require("./parsers");
+const core = require("./core");
+const consts = require("./consts.json");
+
+/**
+ * Maps parameters passed to Autocomplete function to an object with parsed values.
+ *
+ * @param params: [ { value: any, name: string, type: string } ]
+ * Array of values passed to autocomplete function as action parameters.
+ *
+ * @returns {*} - object with params names as keys and parsed values
+ */
+function parseActionParameters(params) {
+  return params.reduce((acc, {
+    value, name, type, valueType,
+  }) => ({
+    ...acc,
+    [name]: parsers.resolveParser(type || valueType)(value),
+  }), {});
+}
+
+function handleInput(awsService, actionParams, pluginSettings) {
```

Comment:
`handle` is very vague. How is the input handled? Is `awsService` really part of the input?

Reply:
That's a very similar case to the handleInput in core.js. True, awsService is not an input per se, however it does influence how the input will be parsed. I will think of an alternative way of taking care of that and will discuss it with you on call.